vllm/config/observability.py

```diff
@@ -114,6 +114,24 @@ def _validate_collect_detailed_traces(
             value = cast(list[DetailedTraceModules], value[0].split(","))
         return value
 
+    mfu_analysis_mode: str = "fast"
+    """Setting `mfu_analysis_mode` to
+       - "detailed" will capture the torch FX graph and attempt to properly
+         count every flop and byte moved.  This is a very slow operation and
+         should be run infrequently for advanced optimization usecases.
+       - "fast" (default) will attempt to use the 2AND approximation by
+         automatically deriving active parameter count.
+       - "manual" will require user specified `mfu_analysis_active_parameters`
+         and also uses 2AND to approximate MFU.
+         N = number of parameters, D = number of new tokens."""
```

Comment:
Necessary for proper rendering in `--help`

```suggestion
    """Setting `mfu_analysis_mode` to"

       - "detailed" will capture the torch FX graph and attempt to properly
         count every flop and byte moved.  This is a very slow operation and
         should be run infrequently for advanced optimization usecases.\n
       - "fast" (default) will attempt to use the 2AND approximation by
         automatically deriving active parameter count.\n
       - "manual" will require user specified `mfu_analysis_active_parameters`
         and also uses 2AND to approximate MFU.
         N = number of parameters, D = number of new tokens."""
```